src/js/nav/globalNav.js

```diff
@@ -33,7 +33,8 @@ async function getRoutesForApp(app, masterConfig) {
             return (await calculateVisibility(app, subItem, visibility)) && ({
                 ...subItem.title ? {
                     id: subItem.id || '',
-                    title: subItem.title
+                    title: subItem.title,
+                    ignoreCase: subItem.ignoreCase || false
```

Comment:
I don't think we need this shorthand since `undefined` is treated as `false` as well.